sim/src/mechanics/walking.rs

```diff
@@ -731,6 +736,14 @@ impl PedState {
             PedState::WaitingForBus(_, _) => unreachable!(),
         }
     }
+
+    fn time_spent_waiting(&self, now: Time) -> Duration {
+        match self {
+            PedState::WaitingToTurn(_, blocked_since)
```

Comment:
Does "WaitingToTurn" mean "waiting to cross a a street"?

Reply:
Yes. "Turn" as in the map_model sense. They'll never be in this state for `SharedSidewalkCorner`s, because all intersection policies immediately grant that. So this is just waiting for a `TurnType::Crosswalk`